### DISCUSSION TITLE
Interactive CLI for setting up eslint in new projects

### DISCUSSION THREAD

**Discussion by danielsvane** [nuxt/nuxt#17581] [Questions]
This might already be implemented, but haven't been able the find it.

When creating a new Vue project with the Vue CLI, I get options about what features to use. I know it doesn't make sense to have exactly the same options in Nuxt, but it would be helpful to have the eslint guided setup.

I have done this manually after creating a Nuxt project with `npx nuxi init nuxt-app`, but I am unsure if Nuxt requires / would benefit from other eslint extensions than the default Vue setup.

Having this done by a Nuxt CLI would remove my doubts, and streamline setting up new projects (I always use linting).

**Comment by TheAlexLichter** [comment]
IMO a default ESLint config definitely makes sense. Compared to the Nuxt2 eslint setup, I'd probably suggest a more lenient config that focuses on fixing errors and allow being combined with any other "style guide" config (like standard, AirBnb, etc. etc.).
